csi-node-driver-registrar/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.13.5 AS build
+ENV GIT_UPSTREAM_REPO=https://github.com/kubernetes-csi/node-driver-registrar
+ENV GIT_VERSION=v1.2.0
+WORKDIR /go/src/github.com/storageos/images/csi-node-driver-registrar
+COPY LICENSE .
+COPY csi-node-driver-registrar/build.sh /
+
+# Clone upstream with full history and checkout to the forked branch to retain
+# correct version tag used in the build flag.
+RUN git clone $GIT_UPSTREAM_REPO $GOPATH/src/github.com/kubernetes-csi/node-driver-registrar
+RUN cd "$GOPATH/src/github.com/kubernetes-csi/node-driver-registrar" \
+    && git checkout $GIT_VERSION \
+    && bash /build.sh
+    # The default make target builds linux and windows binaries together. Use
+    # make when the build server has enough resources.
+    # && make
```

Comment:
Using separate build script is fine here. The version of release-tools at this version builds for windows at the same time https://github.com/kubernetes-csi/node-driver-registrar/blob/v1.2.0/release-tools/build.make#L66 .